identity/src/main/java/androidx/security/identity/GattServer.java

```diff
@@ -329,15 +443,26 @@ public void onNotificationSent(BluetoothDevice device, int status) {
     }
 
     void sendMessage(@NonNull byte[] data) {
+        if ((mLoggingFlags & Constants.LOGGING_FLAG_TRANSPORT_SPECIFIC_VERBOSE) != 0) {
+            Log.i(TAG, "sendMessage " + Util.toHex(data));
+        }
+
+        // Uses socket L2CAP when it is available
+        if (supportL2CAP && mSocket != null && mSocket.isConnected()) {
+            try {
+                mSocket.getOutputStream().write(data);
+                return;
+            } catch (IOException e) {
+                Log.e(TAG, "Error sending message by L2CAP socket " + e.getMessage(), e);
+                reportError(new Error("Error sending message by L2CAP socket"));
+            }
```

Comment:
return here instead of falling through on error?

Reply:
Added `return` after reporting the errror